.vale/styles/RedHat/Spelling.yml

```diff
@@ -98,6 +98,8 @@ filters:
   - "[sS]erializer"
   - "[sS]erverless"
   - "[sS]harding"
+  - "[sS]ysctl"
+  - "[sS]ysctls"
```

Comment:
@kquinn1204 - vale-at-red-hat SMEs confirmed previously in the slack channel that we do not need to add the plural form of each term into the rule file. The tool should handle plurals so I think you can remove line 102.

@aireilly - WDYT?